app/queries/lookup_by_base_paths.rb

```diff
@@ -0,0 +1,57 @@
+module Queries
+  module LookupByBasePaths
+    def self.call(base_paths)
+      base_paths_and_content_ids = Edition.with_document
+        .left_outer_joins(:unpublishing)
+        .left_outer_joins(:access_limit)
+        .where(base_path: base_paths)
+        .where("access_limits.edition_id IS NULL")
```

Comment:
I _think_ we can group these together like `where(base_path: base_paths, "access_limits.edition_id": nil)`